prefs/HBListController.m

```diff
@@ -61,6 +61,69 @@ - (NSMutableArray *)loadSpecifiersFromPlistName:(NSString *)plistName target:(PS
 	return [self _hb_configureSpecifiers:specifiers];
 }
 
+- (NSInteger)imageSystemWeight:(id)weight {
+	if ([weight isKindOfClass:NSNumber.class]) {
+		return [weight integerValue];
+	}
+	if (![weight isKindOfClass:NSString.class]) {
+		return UIImageSymbolWeightRegular;
+	}
+	if ([weight isEqualToString:@"UIImageSymbolWeightUltraLight"] || [weight isEqualToString:@"ultraLight"]) {
+		return UIImageSymbolWeightUltraLight;
+	}
+	if ([weight isEqualToString:@"UIImageSymbolWeightThin"] || [weight isEqualToString:@"thin"]) {
+		return UIImageSymbolWeightThin;
+	}
+	if ([weight isEqualToString:@"UIImageSymbolWeightLight"] || [weight isEqualToString:@"light"]) {
+		return UIImageSymbolWeightLight;
+	}
+	if ([weight isEqualToString:@"UIImageSymbolWeightMedium"] || [weight isEqualToString:@"medium"]) {
+		return UIImageSymbolWeightMedium;
+	}
+	if ([weight isEqualToString:@"UIImageSymbolWeightSemibold"] || [weight isEqualToString:@"semiBold"]) {
+		return UIImageSymbolWeightSemibold;
+	}
+	if ([weight isEqualToString:@"UIImageSymbolWeightBold"] || [weight isEqualToString:@"bold"]) {
+		return UIImageSymbolWeightBold;
+	}
+	if ([weight isEqualToString:@"UIImageSymbolWeightHeavy"] || [weight isEqualToString:@"heavy"]) {
+		return UIImageSymbolWeightHeavy;
+	}
+	if ([weight isEqualToString:@"UIImageSymbolWeightBlack"] || [weight isEqualToString:@"black"]) {
+		return UIImageSymbolWeightBlack;
+	}
+	return UIImageSymbolWeightRegular;
+}
+
+- (NSInteger)imageSystemScale:(id)scale {
+	if ([scaleValue isKindOfClass:NSNumber.class]) {
+		return [scaleValue integerValue];
+	}
+	if (![scaleValue isKindOfClass:NSString.class]) {
+		return UIImageSymbolScaleMedium;
+	}
+	if ([scale isEqualToString:@"UIImageSymbolScaleSmall"] || [scale isEqualToString:@"small"]) {
+		return UIImageSymbolScaleSmall;
+	}
+	if ([scale isEqualToString:@"UIImageSymbolScaleMedium"] || [scale isEqualToString:@"medium"]) {
+		return UIImageSymbolScaleMedium;
+	}
+	if ([scale isEqualToString:@"UIImageSymbolScaleLarge"] || [scale isEqualToString:@"large"]) {
+		return UIImageSymbolScaleLarge;
+	}
+	return UIImageSymbolScaleMedium;
+}
+
+- (UIImage *)imageSystemFromDict:(NSDictionary *)imageSystem {
+	UIImageSymbolConfiguration *configuration = [UIImageSymbolConfiguration
+		configurationWithPointSize:([imageSystem[@"pointSize"] floatValue] ?: 20.0)
+		weight:[self imageSystemWeight:weight]
+		scale:[self imageSystemScale:scale]
+	];
```

Comment:
Move this `];` to the end of the previous line

Reply:
ew